src/main/resources/application.properties

```diff
@@ -1,13 +1,17 @@
+#Sovereign cloud configuration
+authorityEndpointHost="https://login.microsoftonline.com/"
+msGraphEndpointHost="https://graph.microsoft.com/"
```

Comment:
You don't need the apps.json values here. You just need to add a MS Graph endpoint in the config which should be used in the code.